BitwardenShared/UI/Vault/VaultItem/CipherItemState.swift

```diff
@@ -107,6 +107,18 @@ struct CipherItemState: Equatable {
         self
     }
 
+    /// Whether or not this item can be assign to collections.
+    var canAssigntoCollection: Bool {
```

Comment:
⛏️ Capital letter typo:
```suggestion
    var canAssignToCollection: Bool {
```